demo/src/examples/12 Load and Save.svelte

```diff
@@ -25,10 +24,15 @@
 
     map.on("load", () => {
       directions = new CustomMapLibreGlDirections(map);
-      directions.interactive = interactive;
+      directions.interactive = true;
     });
   });
 
+  const noDataToLoad =
+    !localStorage.getItem("saved-waypoints-features") ||
+    !localStorage.getItem("saved-snappoints-features") ||
+    !localStorage.getItem("saved-routelines-features");
+
```

Comment:
In incognito mode on chrome, I can click on Load but nothing appears on the map. Is this here maybe not the correct way to check if there is some data?

Reply:
@wipfli, are you sure you checked out the correct branch? I can't reproduce the issue :confused: 

![Screenshot_20220714_124343](https://user-images.githubusercontent.com/9782236/178941568-ea6193c1-33c0-47b6-aee0-d5d3a27da8a5.png)